google/cloud/bigtable/tests/data_integration_test.cc

```diff
@@ -587,3 +587,51 @@ TEST_F(DataIntegrationTest, TableSampleRowKeysTest) {
   auto last = samples.back();
   EXPECT_LT(0, last.offset_bytes);
 }
+
+TEST_F(DataIntegrationTest, TableReadMulipleCellsBigValue) {
+  if (UsingCloudBigtableEmulator()) {
+    // TODO(#151) - remove workarounds for emulator bug(s).
+    return;
+  }
+
+  std::string const table_id = RandomTableId();
+  auto table = CreateTable(table_id, table_config);
+
+  std::string const row_key = "row-key-1";
+  // cell vector contains 8 cell of ~32 MiB
+  auto const MiB = 1024L * 1024L;
+
+  std::string value((32 * MiB) - 1024, 'a');
+  std::vector<bigtable::Cell> created;
+
+  for (int i = 0; i < 8; i++) {
+    std::string col_qualifiery = "c" + std::to_string(i);
+    created.push_back(
+        bigtable::Cell(row_key, family, col_qualifiery, 0, value, {}));
+  }
+
+  std::vector<bigtable::Cell> expected;
+  for (int i = 0; i < 8; i++) {
```

Comment:
This for loop is identical to the one above. Why not create the two vectors in the same for loop if they're intended to be identical?